### ISSUE TITLE
Having `@nuxt/image` dependency installed throws error on Nuxt restart (in dev mode)

### ISSUE THREAD

**Issue by BobbieGoede** [nuxt-modules/og-image#214]
It's a weird issue and I'm not sure how to debug this, I have a reproduction that only works locally (not in stackblitz) on both my Macbook and on my Windows in WSL. https://stackblitz.com/edit/github-sv9mcq?file=nuxt.config.ts

Working as expected:
1. Run `npm install` and start dev server with `npm run dev`
2. Trigger Nuxt restart by editing and saving `nuxt.config.ts`,
3. This should work as expected

Steps to reproduce issue:
1. Install `@nuxt/image` with `npm install -D @nuxt/image` (no need to add to `nuxt.config.ts`)
2. Start dev server with `npm run dev`
3. Trigger Nuxt restart by editing and saving `nuxt.config.ts`,
4. Throws the following error in logs:
```sh
ERROR  [worker reload] [worker init]                                                                                                                                                        
Something went wrong installing the "sharp" module
```

Possibly related to #158?

**Comment by narr07** [comment]
Same issue

**Comment by harlan-zw** [comment]
This should be fixed in 3.0.0-rc.54. I wasn't able to replicate it locally on my Linux env though.

**Comment by BobbieGoede** [comment]
I can confirm that updating to `3.0.0-rc.54` resolved the issue in my project, thanks!
